trace/trace.go

```diff
@@ -125,6 +128,8 @@ func StartSpan(ctx context.Context, name string) (context.Context, *Span) {
 // StartSpanWithOptions starts a new child span of the current span in the context.
 //
 // If there is no span in the context, creates a new trace and span.
+//
+// Deprecated: use StartSpan(...), or WithSpan(ctx, NewSpan(...)).
```

Comment:
Deprecated: Use StartSpan or WithSpan.

Typical punctuation rules. Start with capital after `:` if it is a sentence.

We refer to public identifiers with their name only.

Reply:
TIL!